static/js/message_edit.js

```diff
@@ -579,6 +579,27 @@ exports.delete_message = function (msg_id) {
     });
 };
 
+exports.report_spam = function (msg_id) {
```

Comment:
Not sure this file is the right place to put this logic, thoughts?

Reply:
@shubhamdhama I really don't know, do you have a better option where to place it?